infra/api/service/main.tf

```diff
@@ -138,8 +138,26 @@ module "service" {
     }
   } : null
 
-  extra_environment_variables = merge(local.service_config.extra_environment_variables, { "ENVIRONMENT" : var.environment_name })
```

Comment:
It looks like you undid the change you made here

Reply:
Ahah! Thanks